clouddriver-cf/src/main/groovy/com/netflix/spinnaker/clouddriver/cf/model/CloudFoundryLoadBalancerProvider.groovy

```diff
@@ -29,34 +29,28 @@ class CloudFoundryLoadBalancerProvider implements LoadBalancerProvider<CloudFoun
   @Autowired
   CloudFoundryResourceRetriever cloudFoundryResourceRetriever
 
-  @Override
   Map<String, Set<CloudFoundryLoadBalancer>> getLoadBalancers() {
```

Comment:
Any objection to just removing the implementations (or @gregturn plan to do that as a followup? )